src/object.rs

```diff
@@ -755,6 +767,45 @@ impl Object {
     }
     .into()
   }
+
+  /// Gets the property attributes of a property which can be [None] or any
+  /// combination of [PropertyAttribute::READ_ONLY],
+  /// [PropertyAttribute::DONT_ENUM] and [PropertyAttribute::DONT_DELETE].
+  /// Returns [None] when the property doesn't exist.
+  pub fn get_property_attributes(
+    &self,
+    scope: &mut HandleScope,
+    key: Local<Value>,
+  ) -> Option<PropertyAttribute> {
+    let mut out = Maybe::<PropertyAttribute>::default();
+    unsafe {
+      v8__Object__GetPropertyAttributes(
+        self,
+        &*scope.get_current_context(),
+        &*key,
+        &mut out,
+      )
+    };
+    out.into()
+  }
+
+  /// Implements Object.getOwnPropertyDescriptor(O, P), see
+  /// https://tc39.es/ecma262/#sec-object.getownpropertydescriptor.
+  pub fn get_own_property_descriptor<'s>(
```

Comment:
Any chance you could test a code path that returns `None`?

Reply:
V8 seems to never return an Empty MaybeLocal here. They also don't test this in their test_api.cc.